README.md

```diff
@@ -29,7 +29,7 @@ We support integration using CocoaPods as well. You can add Programmable Video t
 ```
 source 'https://github.com/CocoaPods/Specs'
 
-platform :ios, '12.0'
+platform :ios, '11.0'
```

Comment:
Also bump the min SDK version to 4.1.

Reply:
Ah.. We need to make sure we do that as part of the release process....

We are already publishing the package.swift file to this repo... We could also make it update the README in an automated fashion. I will work on that tomorrow me thinks.

Reply:
That would definitely be useful for consistency. In this case I was most worried about the iOS version not being satisfied by 4.0.x.